owasp-top10-2017-apps/a7/gossip-world/app/routes.py

```diff
@@ -139,7 +140,7 @@ def newuser():
 @app.route('/gossip', methods=['GET'])
 @login_required
 def all_gossips():
-    search = request.args.get('search')
+    search = escape(request.args.get('search'))
     search_flag = 0
     if search is not None:
```

Comment:
When `search` is _None_, `escape` changes it to _string_ and this validation doesn't work as expected.

![captura de tela 2019-02-08 as 16 19 48](https://user-images.githubusercontent.com/17754098/52497677-cd054a00-2bbd-11e9-90b3-a8058e717b73.png)

You could escape search variable after the validation, what do you think?

Reply:
Done. Thanks you